Apps/Admin/Tests/Services/CsvExportServiceTests.cs

```diff
@@ -133,7 +133,6 @@ public void TestMapperAdminUserProfileViewFromUserRepresentation()
             {
                 AdminUserProfileId = null,
                 UserId = userRepresentation.UserId,
-                LastLoginDateTime = null,
```

Comment:
Why is this test being changed? This is not related to the support page. It is dealing with System Analytics; looks like keycloak to Admin user profile view.  Inactive users report.